yt_dlp/utils.py

```diff
@@ -840,11 +840,17 @@ class Popen(subprocess.Popen):
     else:
         _startupinfo = None
 
-    def __init__(self, *args, text=False, **kwargs):
+    def __init__(self, *args, env=None, text=False, **kwargs):
         if text is True:
             kwargs['universal_newlines'] = True  # For 3.6 compatibility
             kwargs.setdefault('encoding', 'utf-8')
             kwargs.setdefault('errors', 'replace')
+        if sys.platform != 'win32':
```

Comment:
`if sys.platform != 'win32'` will be true for cygwin. I'm not sure if that's the intended behaviour

Reply:
oops

Reply:
@Lesmiscore also, according to [this](https://stackoverflow.com/questions/446209/possible-values-from-sys-platform) (top answer), msys2 (whatever that is) also has it's own sys.platform value